Gruntfile.js

```diff
@@ -93,6 +93,7 @@ module.exports = function(grunt) {
   ]);
   publishTask('test', 'Run unittests', ['run:test']);
   publishTask('serve', 'Serve out/ on 127.0.0.1:8080', ['http-server:.']);
+  // Why isn't this one using publishTask?
```

Comment:
publishTask creates a task, publishTasks.push just adds an existing one to the help message.

Honestly the amount of work I've done here with grunt is kind of dumb, but I think it makes it easier to use.